### PR TITLE
Add some missing attributes

### PR DESCRIPTION
**Pull request by aleokdev**
Add some inline & crate attributes.

### CODE REVIEW

**Review by bjorn — APPROVED** [review]
Approved, but I had one question.

File: src/layers/image.rs [hunk 1]
```diff
@@ -39,6 +39,7 @@ map_wrapper!(ImageLayer => ImageLayerData);
 
 impl<'map> ImageLayer<'map> {
     /// Get a reference to the image layer's image.
+    #[inline]
```
[the remark this answers is not in the document]
Comment (aleokdev) [reply src/layers/image.rs:42]: Yes, but it's useful to hint it. Same thing as in C++ code.

[the remark this answers is not in the document]
Comment (bjorn) [reply src/layers/image.rs:42]: Well, seems it's [different in Rust](https://matklad.github.io/2021/07/09/inline-in-rust.html) actually:

> Without #[inline], even the most trivial of functions can’t be inlined across the crate boundary.

(when not compiling with LTO). This results in the following recommendation:

> when building libraries, proactively add #[inline] to small non-generic functions

So yeah, let's do that. :-)

**Review by bjorn — APPROVED** [review]

**Review by bjorn — APPROVED** [review]

**Comment by aleokdev** [comment]
image
Thank god we have CI set up lmao

**Review by bjorn — APPROVED** [review]

**Pull request merged**